core/arch/arm/plat-poplar/conf.mk

```diff
@@ -29,4 +29,6 @@ CFG_SECURE_DATA_PATH ?= y
 CFG_TEE_SDP_MEM_BASE ?= 0x02800000
 CFG_TEE_SDP_MEM_SIZE ?= 0x00400000
 
+CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
```

Comment:
Any reason why you're not assigning `CFG_ENABLE_EMBEDDED_TESTS` instead?
With that you'd catch other future embedded tests too.

Reply:
Not really, I followed the same way as it's done for other boards:

```
$ grep -e CFG_TEE_CORE_EMBED_INTERNAL_TESTS -r ./core/arch/arm/plat-*
./core/arch/arm/plat-rpi3/conf.mk:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
./core/arch/arm/plat-stm/conf.mk:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
./core/arch/arm/plat-stm32mp1/conf.mk:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y


```

Reply:
So probably it makes sense to `s/CFG_TEE_CORE_EMBED_INTERNAL_TESTS/CFG_ENABLE_EMBEDDED_TESTS/g` for other boards

Reply:
I should have mentioned this earlier.
The reason we have this default off for everything else is to minimize the risk that debug or test code is unintentionally included in a product. So you may want to set `CFG_ENABLE_EMBEDDED_TESTS=y` in the build script you use or so.
With that in mind, if you really want it here I suppose that's OK too.

Reply:
Got it. Let's do that differently then.
Please ignore this PR, and I'll add `CFG_ENABLE_EMBEDDED_TESTS=y` to https://github.com/OP-TEE/build/blob/master/poplar.mk. We will kill two birds with one stone: make xtest running without errors (in our default buildroot setup) and avoid default enabling these tests in OP-TEE core

Reply:
Sound good.